.cirrus.yml

```diff
@@ -140,23 +140,24 @@ task:
   matrix:
     - name: Linux aarch64
       arm_container:
-        image: rust:1.56
+        image: rust:1.63
       env:
         RUSTFLAGS: --cfg graviton -D warnings
         TARGET: aarch64-unknown-linux-gnu
     - name: Linux x86_64
       container:
-        image: rust:1.56
+        image: rust:1.63
       env:
         TARGET: x86_64-unknown-linux-gnu
     - name: Linux x86_64 musl
       container:
-        image: rust:1.56
+        image: rust:1.63
       env:
         TARGET: x86_64-unknown-linux-musl
   setup_script:
     - rustup target add $TARGET
-    - rustup component add clippy
+    - rustup toolchain install $TOOLCHAIN --profile minimal --target $TARGET
```

Comment:
Why is this step necessary?

Reply:
I copied this from the other `setup_script` blocks. If you remove this I'm pretty sure the build fails with a wrong rust version error.

Reply:
And yet the old version didn't fail.  I think this section is masking some other problem.